### PR TITLE
chore!: bumped aplib to latest and applied current constructor norms

### PR DESCRIPTION
**Pull request by SilasPeters**

### CODE REVIEW

File: Runtime/Aplib/Integrations/Unity/Actions/UnityPathfinderAction.cs [hunk 1]
```diff
@@ -49,7 +89,7 @@ public class UnityPathfinderAction<TBeliefSet> : Core.Intent.Actions.Action<TBel
 
             if (path.corners.Length <= 1) return;
 
-            // Move Towards
+            // Calculate and indicate in what direction we should move
```
Comment (QuakeEye) [inline Runtime/Aplib/Integrations/Unity/Actions/UnityPathfinderAction.cs]:
```suggestion
            // Calculate and indicate in what direction we should move.
```

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "com.team-zomsa.aplib-unity",
-    "version": "1.1.0",
+    "version": "2.0.0",
```
Comment (QuakeEye) [inline package.json]: Would make this the same as the Aplib.Net version that is within the package. (Right now seems to be v1.11.0-dev.6 ?)

**Review by QuakeEye — CHANGES_REQUESTED** [review]

**Review by joachimdekker — APPROVED** [review]
Looks good!

**Review by QuakeEye — APPROVED** [review]
image

**Pull request merged**